src/new-user-experience/modules/advanced-settings/index.js

```diff
@@ -30,84 +30,86 @@ export const AdvancedSettings = () => {
 	const { enable_customer_decline_messages, debug_logging_enabled = 'no' } =
 		settings;
 
-	if (!squareSettingsLoaded) {
+	if ( ! squareSettingsLoaded ) {
 		return null;
 	}
 
 	return (
 		<>
 			<Section>
 				<SectionTitle
-					title={__('Advanced Settings', 'woocommerce-square')}
+					title={ __( 'Advanced Settings', 'woocommerce' ) }
```

Comment:
Looks like we're changing all the textdomains here (and in other files). Curious for the reason?

Reply:
It's the `npm run lint:js:fix` command, I am not sure why it is replacing the text domain.

Reply:
For future ref: https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/eslint-plugin/configs/recommended.js#L24